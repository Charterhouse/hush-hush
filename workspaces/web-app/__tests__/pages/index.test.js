import React from 'react'
import { render } from 'react-testing-library'
import Home from '../../pages/index'

describe('index', () => {
  it('has proper styling', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })

  it('displays secret form', () => {
    const { getByText } = render(<Home />)
    expect(getByText(/your secret/i)).toBeInTheDocument()
  })
})
