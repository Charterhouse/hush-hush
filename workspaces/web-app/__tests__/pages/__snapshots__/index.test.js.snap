// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index has proper styling 1`] = `
.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100vh;
  width: 100vw;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
}

.emotion-0 {
  font-family: "Roboto Mono",monospace;
  font-size: 1.2em;
  margin-bottom: 10px;
  opacity: 0.5;
}

.emotion-2 {
  font-family: "Roboto Mono",monospace;
  width: 100%;
  max-width: 500px;
  background-color: black;
  color: white;
  border-radius: 20px;
  border: 1px solid white;
  padding: 20px;
  resize: none;
  outline-width: 0;
  -webkit-transition: border-color 0.2s ease-in-out 0s;
  transition: border-color 0.2s ease-in-out 0s;
}

.emotion-2:focus {
  border-color: #0099FF;
}

.emotion-2::selection {
  background-color: white;
}

.emotion-6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  opacity: 1;
  -webkit-transition: opacity 1s ease-in-out 0s;
  transition: opacity 1s ease-in-out 0s;
}

<div>
  <div
    class="emotion-8 emotion-9"
  >
    <div
      css="[object Object]"
    >
      <div
        class="emotion-6 emotion-7"
        opacity="1"
      >
        <form
          class="emotion-4 emotion-5"
        >
          <label
            class="emotion-0 emotion-1"
            for="frmEmailA"
          >
            Recipient:
          </label>
          <input
            class="emotion-2 emotion-3"
            id="frmEmailA"
            name="email"
            placeholder="name@example.com"
            required=""
            type="email"
            value=""
          />
        </form>
      </div>
    </div>
  </div>
</div>
`;
